container-startup

```diff
@@ -5,15 +5,15 @@
 CMD=$1
 shift
 
-function usage() {
+usage() {
 	echo "pcw startup helper"
 	echo "Usage: $0 COMMAND"
 	echo "COMMANDS:"
 	echo "   help                    Print help message"
 	echo "   check                   Run system health checks"
 	echo "   run                     Standart publiccloud-watcher run procedure"
 	echo "   migrate                 Run database migration"
-	echo "   createsuperuser         Create superuser. Requires --email and --username argument"
+	echo "   createuser USER PASS    Create user"
```

Comment:
if idea to have only single user why do we need to have special command to add users ?

Reply:
Because otherwise we would have to run `python3 manage.py shell -c "from django.contrib.auth.models import User; user = User.objects.create_user('USERNAME', password='PASSWORD'); user.save()"` which is ugly.

Reply:
no I mean we already have "createsuperuser" and we don't want to have other users at all so why do we need to have "createuser" at all ? who and when will call it ?

Reply:
> no I mean we already have "createsuperuser" and we don't want to have other users at all so why do we need to have "createuser" at all ? who and when will call it ?

The problem with `createsuperuser` is that it creates a special Django superuser which we don't need at all.  A normal user as created by `createuser` is enough.  The Django superuser is needed only for the Django admin interface that we don't use.